tool/public/client.js

```diff
@@ -125,7 +134,9 @@ class FlagConverter {
 					// region Discard these
 					case "srd5e":
 					case "core":
-					case "favtab": // https://github.com/syl3r86/favtab
+					case "favtab": // https://foundryvtt.com/packages/favtab
+					case "magicitems": // https://foundryvtt.com/packages/magicitems
+					case "cf": // (unknown)
```

Comment:
any idea what this is from? it was on your Fey Ancestry feature, I believe

Reply:
Looking at the data it has (`id` and `color`), my guess is [Compendium Folders](https://github.com/earlSt1/vtt-compendium-folders). No guarantee for that yet (I can't even remember if I still use that module 🤣)